invenio_communities/templates/semantic-ui/invenio_notifications/community-invitation.accept.jinja

```diff
@@ -2,42 +2,70 @@
 {% set receiver = invitation_request.receiver %}
 {% set community = invitation_request.created_by %}
 {% set request_id = invitation_request.id %}
-
+{% set message = notification.context.message | safe if notification.context.message else '' %}
 {% set community_title = community.metadata.title %}
-{% set receiver_name = receiver.profile.full_name or receiver.username %}
+{% set receiver_name = receiver.username or receiver.profile.full_name %}
 
 {# TODO: use request.links.self_html when issue issue is resolved: https://github.com/inveniosoftware/invenio-rdm-records/issues/1327 #}
 {% set request_link = "{ui}/me/requests/{id}".format(
     ui=config.SITE_UI_URL, id=request_id
     )
 %}
+{% set account_settings_link = "{ui}/account/settings/notifications".format(
+    ui=config.SITE_UI_URL
+    )
+%}
 
 {%- block subject -%}
-  {{ _("Community member invitation accepted") }}
+   {{ _("✅ @{user_name} accepted the invitation to join community '{community_title}'.").format(user_name=receiver_name, community_title=community_title) }}
 {%- endblock subject -%}
 
 {%- block html_body -%}
-<p>
-    {{ _("'{user_name}' accepted the invitation to join community '{community_title}'.").format(user_name=receiver_name, community_title=community_title) }}
-</p>
-
-<a href="{{ request_link }}" class="button"> {{ _("Check out the invitation") }}</a>
+<table style="font-family:'Lato',Helvetica,Arial,sans-serif;border-spacing:15px">
+    <tr>
+        <td>{{ _("<b>@{user_name}</b> accepted the invitation to join your community <b>{community_title}</b>").format(user_name=receiver_name,community_title=community_title) }}
```

Comment:
ping @rekt-hard I think different languages had other email templates ? correct me if I am wrong please. In this case we probably dont need to use lazystring